### DISCUSSION TITLE
Enabling column reorder

### DISCUSSION THREAD

**Discussion by pythonquick** [6pac/SlickGrid#1019] [Q&A] [answered]
When setting up the SlickGrid instance, the following error shows up:
`ReferenceError: Sortable is not defined
 at SlickGrid.initialize (slick.grid.ts:659:48)
`
unless i apply the SlickGrid option `enableColumnReorder: false`

I do need the column reorder function. A workaround is to separately import `Sortable` from the `sortablejs` module into global scope. 
Is there a better way to enable column reorder?

**Comment by ghiscoding** [comment, marked as answer]
Sortable is the only dependency of SlickGrid and it must be loaded before SlickGrid itself, if you use the IIFE build then simply move your Sortable `<script>` import before SlickGrid, or if you use other build (CJS/ESM) then assign Sortable to the `window` or `global` object. It's already mentioned in the [Migration Guide to 5.0](https://github.com/6pac/SlickGrid/wiki/Major-version-5.0-%E2%80%90-ES6-ESM-and-TypeScript-Support#es6--esm-cjs-esm)
